s3tests_boto3/functional/test_s3.py

```diff
@@ -632,6 +986,17 @@ def test_bucket_list_maxkeys_none():
     eq(keys, key_names)
     eq(response['MaxKeys'], 1000)
 
+def test_bucket_listv2_maxkeys_none():
+    key_names = ['bar', 'baz', 'foo', 'quxx']
+    bucket_name = _create_objects(keys=key_names)
+    client = get_client()
+
+    response = client.list_objects_v2(Bucket=bucket_name)
+    eq(response['IsTruncated'], False)
+    keys = _get_keys(response)
+    eq(keys, key_names)
+    eq(response['MaxKeys'], 1000)
```

Comment:
When MaxKeys is not specified, it defaults to 1000. Am I understanding this right?